src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3595,6 +3595,93 @@
     public static final int RECORD_DEF =
             GeneratedJavaTokenTypes.RECORD_DEF;
 
+    /**
+     * Beginning of a Java 14 Text Block literal,
+     * delimited by three double quotes.
+     *
+     * <p>For example:</p>
+     * <pre>
+     *         String hello = """
+     *                 Hello, world!
+     *                 """;
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * |--VARIABLE_DEF
+     * |   |--MODIFIERS
+     * |   |--TYPE
+     * |   |   `--IDENT (String)
+     * |   |--IDENT (hello)
+     * |   |--ASSIGN (=)
+     * |   |   `--EXPR
+     * |   |       `--TEXT_BLOCK_LITERAL_BEGIN (""")
+     * |   |           |--TEXT_BLOCK_CONTENT (\n                Hello, world!\n                    )
+     * |   |           `--TEXT_BLOCK_LITERAL_END (""")
+     * |   `--SEMI (;)
+     * </pre>
+     *
+     */
```

Comment:
Should `@since` tag be here? We agreed to tag new tokens.

Reply:
Updated.